ynput/ayon-houdini
# Add publish families to cache creators

## Changelog Description
This PR 
- adds  publish families to cache creators.
- removes unnecessary collector.
- implements callbacks to cachesize collector to make its attribute visible if `farm`  is enabled

> [!Note]
> This PR is mainly made for sharing code and show a use case  for https://github.com/ynput/ayon-core/pull/994
> It should be used along https://github.com/ynput/ayon-deadline/pull/49 and https://github.com/ynput/ayon-deadline/pull/54
> Here's what to expect when using the three PRs togther
> ![Animation_17](https://github.com/user-attachments/assets/ca34383e-1a40-4926-8487-d8a584be2512)

## Testing notes:
1. Launch Houdini via AYON
2. Create a cache instance (vdb, model, pointcache, ass, rsproxy)
3. Switch farm toggle in the publisher UI, click save,  click refresh and watch the attributes appear and disappear.
